services/ipfs-service.ts

```diff
@@ -10,14 +10,14 @@ const port = serverRuntimeConfig?.infura?.port || "5001";
 const auth =
   "Basic " +
   Buffer.from(serverRuntimeConfig?.infura?.projectId +
-      ":" +
-      serverRuntimeConfig?.infura?.projectSecret).toString("base64");
+    ":" +
+    serverRuntimeConfig?.infura?.projectSecret).toString("base64");
 const baseURL = `https://${host}:${port}/api/v0`;
 
 export async function add(file: Buffer | string,
-                          pin = false,
-                          originalFilename?: string,
-                          ext?: string): Promise<{ hash: string; fileName: string; size: string }> {
+  pin = false,
```

Comment:
lint should scream bloody murder, as arguments should be aligned with the first argument :o